zoologist/theme.json

```diff
@@ -78,10 +78,10 @@
 				}
 			},
 			"color": {
-				"foreground": "var(--wp--preset--color--primary)",
+				"foreground": "var(--wp--preset--color--foreground)",
 				"background": "var(--wp--preset--color--background)",
 				"primary": "var(--wp--preset--color--primary)",
-				"secondary": "var(--wp--preset--color--primary)",
+				"secondary": "var(--wp--preset--color--secondary)",
```

Comment:
I think quadrat and geologist needs the same change

Reply:
Seems like there are a few of these - i'll go through and create some PRs to fix the ones I see.